src/routes/webhooks/customerio.ts

```diff
@@ -153,6 +164,42 @@ export const customerio = async (fastify: FastifyInstance): Promise<void> => {
     { prefix: '/marketing_cta' },
   );
 
+  fastify.post<PromotedPostPayload>('/promote', {
```

Comment:
To make it more specific

```suggestion
  fastify.post<PromotedPostPayload>('/promotePost', {
```